platform/viewer/cypress/integration/common/OHIFLoadLocalFile.spec.js

```diff
@@ -0,0 +1,31 @@
+describe('OHIF Load Local File', () => {
+  beforeEach(() => {
+    cy.visit('/local');
+  });
+
+  it('checks if user can navigate to /local', () => {
+    cy.get('.drag-drop-contents').should(
+      'contain',
+      'Drag and Drop DICOM files here to load them in the Viewer'
+    );
+  });
+
+  it('loads an invalid file and verify if viewer is empty', () => {
+    const fileName = 'example.json';
+
+    cy.fixture(fileName).then(fileContent => {
+      cy.get('input[type=file]')
+        .first()
+        .upload({
+          fileContent,
+          fileName,
+          mimeType: 'application/json',
+        });
+    });
+
+    //Verify if there is no thumbnail visible
+    cy.expectMinimumThumbnails(0);
+    //Verify if DOC thumnails is displayed and load it on viewport
+    cy.get('[data-cy="thumbnail-list"]').should('not.exist');
+  });
+});
```

Comment:
Interesting test. Would we want to show a warning or prevent this behavior for invalid files? Am I right to  assume that the viewer route shows, but with no data to display?